app/Http/Controllers/WebhookController.php

```diff
@@ -99,9 +105,15 @@ public function update(WebhookUpdateRequest $request, Webhook $webhook)
         try {
             $webhook = $this->webhookRepository->update($webhook->id, $data);
             return redirect()->route('webhooks.edit', $webhook)
-                ->with('messageSuccess', 'This webhook successfully updated');
+                ->with('messageSuccess', [
+                    'status' => 'Update success',
+                    'message' => 'This webhook successfully updated'
+                ]);
         } catch (QueryException $exception) {
-            return redirect()->back()->with('messageFail', 'Update failed. Something went wrong')->withInput();
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Update failed',
+                'message' => 'Update failed. Something went wrong'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 115</br>Each line in an array declaration must end in a comma